events.yaml

```diff
@@ -0,0 +1,35 @@
+x-events:
+  - name: cancel_install_failed_start
+    id: "CIE00001"
+    message: "Failed to cancel installation: error starting DB transaction"
+    event_type: cluster
+    severity: "error"
+    properties:
+        cluster_id:  UUID
+
+  - name: cancel_install_failed_commit
+    id: "CIE00002"
+    message: "Failed to cancel installation: error committing DB transaction"
+    event_type: cluster
```

Comment:
what are the options for the event_type? is it an enum?

Reply:
event type is used for validation.
And events code is generated slightly different